sdk/storage/storage-file-datalake/README.md

```diff
@@ -178,7 +190,9 @@ Use the constructor to create a instance of `DataLakeServiceClient`.
 
   [Note - Above steps are only for Node.js]
 
-- Alternatively, you instantiate a `DataLakeServiceClient` with a `StorageSharedKeyCredential` by passing account-name and account-key as arguments. (account-name and account-key can be obtained from the azure portal)
+#### with StorageSharedKeyCredential
```

Comment:
what about `StorageSharedKeyCredential`?

Reply:
The headers combine to a whole sentence: `Create the XX service client, with xx credential`.

Reply:
It's formated, What about enclose StorageSharedKeyCredential '`' considering it's a type in code?

Reply:
Sure, fixed.